src/misc/applause.ts

```diff
@@ -0,0 +1,58 @@
+import { SoundCategory } from 'org.bukkit';
+import { Player } from 'org.bukkit.entity';
+
+// Applause
+registerCommand(
+  ['applause', 'aplodit'],
+  (sender) => {
+    if (sender instanceof Player) {
+      clap(sender, 10, 0.2);
+    }
+  },
+  {
+    executableBy: 'players',
+    accessChecker: () => true,
```

Comment:
Instead of defining a custom access checker, we should *prefer* using permissions and then just giving them to all players.